src/chart/summaryTable/renderRow/makeBarChart.js

```diff
@@ -0,0 +1,152 @@
+import clone from '../../../histogram/util/clone';
+
+export function makeBarChart(this_, d) {
+    const chartContainer = d3.select(this_).node();
+    const chartSettings =
+        {x: {column: 'prop_n'
+            ,type: 'linear'
+            ,label: ''
+            ,format: '%'
+            ,domain: [0,null]}
+        ,y: {column: 'key'
+            ,type: 'ordinal'
+            ,label: ''}
+        ,marks:
+            [
+                {type: 'bar'
+                ,per: ['key']
+                ,summarizeX: 'mean'
+                ,tooltip: '[key]: [n] ([prop_n])'}
+            ]
+        ,gridlines: 'xy'
+        ,resizable: false
+        ,height: this_.height
+        ,margin: this_.margin
+        ,value_col: d.value_col
+        };
+
+    function onInit() {
+      //Add group labels.
+        if (this.config.group) {
+            const groupTitle = this.wrap
+                .append('p')
+                .text(`${this.config.group_col}: ${this.config.group} (n=${this.config.n})`);
+            this.wrap.node().parentNode
+                .insertBefore(groupTitle.node(), this.wrap.node())
+        }
+    }
+
+    function onResize() {
+      //Move y-axis labels to the right.
+        const ticks = this.wrap
+            .selectAll('g.y.axis g.tick');
+
+        ticks.select('text').remove();
+        ticks.append('title')
+            .text(d => d);
+        ticks
+            .append('text')
+            .attr(
+                {'text-anchor': 'start'
+                ,'alignment-baseline': 'middle'
+                ,'dx': '1em'
+                ,'x': this.plot_width})
+            .text(d => d.length < 30
+                ? d
+                : d.substring(0,30)+'...');
+
+      //Draw overall rates.
+        if (this.config.group) {
+            this.current_data
```

Comment:
So this extra mark is showing the overall rate for the given level *within the context of the selected filters*, right? 

For example, if we are looking at the chart for race (bars are White, Black, etc) grouped by gender (panels for male and female), filtered to a single site (Site 1). The marks for white participants in the females panel would be showing the % of the female population in site 1 that is white (the bar) vs. the total percent of the population (both male and female) in site 1 that is white (the line). Correct?

In other words, the line the example above is the same as the bar in the default example with no group (assuming the same filters)? 

If I'm understanding everything right, I'd suggest keeping that 'overall' bar visually identical for the ungrouped and grouped example. maybe {stroke:black, fill: white, height: x} for the 'overall' mark. and then {fill:black (or some color), height: 0.8*x} for the grouped data]